src/renderer/components/+preferences/__tests__/extensions.test.tsx

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
All of these could and should be covered in behaviours.

Reply:
Removed this file and moved some of the test cases from it to behavioural `src/behaviours/preferences/navigation-to-extension-specific-preferences.test.tsx`